pyseir/deployment/webui_data_adaptor_v1.py

```diff
@@ -312,8 +312,8 @@ def map_fips(self, fips: str) -> None:
             output_model = output_model.fillna(0)
 
             # Fill in results for the Rt indicator.
-            try:
-                rt_results = load_Rt_result(fips)
+            rt_results = load_Rt_result(fips)
+            if rt_results is not None:
```

Comment:
Does this change catch both the ValueError and the KeyError that was being handled previously? I'm curious if that try-except was handling two distinct pieces of logic and now we might have dropped one of them.

Reply:
But in general this is a much more readable way to capture what we were catching. Thank you!

Reply:
That's a good question.  I think I didn't see any evidence of where a value error was actually being raised (and it seems like the latest build didn't throw it).  I would lean towards waiting for it to surface again